strings-csharp/HexToByteArray/HexToByteArray/ConvertHex.cs

```diff
@@ -1,13 +1,15 @@
-﻿namespace HexToByteArray
+﻿using System.Runtime.CompilerServices;
+
```

Comment:
Putting this comment here as it applies across each method.

- The signature can be made consistent across all of them so that each method takes a `ReadOnlySpan<char>`
- Each one can start with the same defensive block:
`if (source.StartsWith("0x")) source = source[2..];`
`if (source.IsEmpty) return Array.Empty<byte>();`
`if (source.Length % 2 != 0) throw new ArgumentException("Invalid hex string", nameof(source));`